cli/cmd/plugin/ui/web/tanzu-ui/src/views/management-cluster/aws/aws-mc-common/management-credential-step/ManagementCredentials.tsx

```diff
@@ -172,30 +159,22 @@ function ManagementCredentials(props: Partial<StepProps>) {
 
     useEffect(() => {
         if (awsState[STORE_SECTION_FORM].REGION) {
-            retrieveOsImages(awsState[STORE_SECTION_FORM].REGION);
-        }
-    }, [awsState[STORE_SECTION_FORM].REGION]);
-
-    function retrieveOsImages(region: string) {
-        try {
-            setOsImages([]);
-            if (region !== '') {
-                AwsService.getAwsosImages(region).then((data) => {
-                    setOsImages(data);
+            AwsOrchestrator.initOsImages({ awsState, awsDispatch })
```

Comment:
please use async await. there is an example in the ManagementCredentialOneTime component.

Reply:
Yes sir. Used `async await`